samples/net/mqtt_publisher/src/main.c

```diff
@@ -107,9 +107,12 @@ static void clear_fds(void)
 
 static void wait(int timeout)
 {
+	int err;
+
 	if (nfds > 0) {
-		if (poll(fds, nfds, timeout) < 0) {
-			printk("poll error: %d\n", errno);
+		err = poll(fds, nfds, timeout);
+		if (err < 0) {
+			printk("poll error: %d\n", err);
```

Comment:
Isn't poll() always returning -1 if there is an error and which is kind of useless to print. It would be much more useful to print the value of errno in this case.

Reply:
oh, errno is a define... I see.. wrong PR